oci/repositories.bzl

```diff
@@ -110,6 +110,43 @@ zot_repositories = repository_rule(
     },
 )
 
+STRUCTURE_TEST_BUILD_TMPL = """\
+# Generated by container/repositories.bzl
+load("@contrib_rules_oci//oci:toolchain.bzl", "structure_test_toolchain")
+structure_test_toolchain(
+    name = "st_toolchain", 
```

Comment:
Don't be afraid of using long labels. `structure_test_toolchain` is much better than `st_toolchain`.